### PR TITLE
Created main structure( index.html,styles.css and accordion.js)

### PR DESCRIPTION
**Pull request by AndriiChuprina**

### CODE REVIEW

File: js-arrays/accordion.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+let acc = document.getElementsByClassName('accordion');
+let i;
+for (i = 0; i < acc.length; i++) {
```
Comment (dmitry-darly-solutions) [inline js-arrays/accordion.js]: for (let i = 0; i < acc.length; i++) {

File: js-arrays/accordion.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    acc[i].addEventListener('click', function() {
```
Comment (dmitry-darly-solutions) [inline js-arrays/accordion.js]: handle click on parent element

File: js-arrays/accordion.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+        this.classList.toggle('active');
+        let panel = this.nextElementSibling;
+        if (panel.style.display === 'block') {
+            panel.style.display = 'none';
```
Comment (dmitry-darly-solutions) [inline js-arrays/accordion.js]: panel.style.display = panel.style.display === 'block' ? 'none' : 'block' ;

File: js-arrays/index.html [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+<h2>Accordion</h2>
+<button class="accordion"> Join all elements</button>
+<div class="panel">
+<input type="text" id="element-join"></input>
+<input type="button" id="button1" value="Add" onclick="addElementToArray();"></input>
+<input type="button" id="button1" value="Separator" onclick="addSeparator();"></input>
```
Comment (dmitry-darly-solutions) [inline js-arrays/index.html]: id should be unique

File: js-arrays/index.html [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+<head>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+<h2>Accordion</h2>
+<button class="accordion"> Join all elements</button>
+<div class="panel">
+<input type="text" id="element-join"></input>
+<input type="button" id="button1" value="Add" onclick="addElementToArray();"></input>
```
Comment (dmitry-darly-solutions) [inline js-arrays/index.html]: use button tag for buttons

File: js-arrays/index.html [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+<input type="button" id="button1" value="Separator" onclick="addSeparator();"></input>
+<input type="button" id="button2" value="Display" onclick="displayArray();"></input>
+</br><span>Result array:</span><span id="result-element-join"></span>
+</div>
+
```
Comment (dmitry-darly-solutions) [inline js-arrays/index.html]: remove empty lines

File: js-arrays/accordion.js [hunk 7]
```diff
@@ -0,0 +1,13 @@
+let acc = document.getElementsByClassName('accordion');
```
Comment (dmitry-darly-solutions) [inline js-arrays/accordion.js]: make sense use id for getting element

File: js-arrays/accordion.js [hunk 8]
```diff
@@ -0,0 +1,11 @@
+document.getElementById('accordionContainer').addEventListener("click",doSmth);
+function doSmth(event) {
+    let target = event.target;
+    if (target.className == 'accordion') {
+        console.dir(target);
```
Comment (dmitry-darly-solutions) [inline js-arrays/accordion.js]: remove console

File: js-arrays/accordion.js [hunk 9]
```diff
@@ -0,0 +1,11 @@
+document.getElementById('accordionContainer').addEventListener("click",doSmth);
```
Comment (dmitry-darly-solutions) [inline js-arrays/accordion.js]: different quotes

File: js-arrays/accordion.js [hunk 10]
```diff
@@ -0,0 +1,11 @@
...
+function doSmth(event) {
```
Comment (dmitry-darly-solutions) [inline js-arrays/accordion.js]: rename function

Comment (dmitry-darly-solutions) [reply js-arrays/accordion.js]: use arrow function

File: js-arrays/accordion.js [hunk 11]
```diff
@@ -0,0 +1,11 @@
...
+    let target = event.target;
+    if (target.className == 'accordion') {
+        console.dir(target);
+        highlight(target.nextElementSibling);
+    }
+}
+function highlight(node) {
```
Comment (dmitry-darly-solutions) [inline js-arrays/accordion.js]: no need function for one string

Comment (AndriiChuprina) [reply js-arrays/accordion.js]: fixed

File: js-arrays/accordion.js (hunk 10 above)
Comment (AndriiChuprina) [reply js-arrays/accordion.js]: fixed

File: js-arrays/accordion.js (hunk 9 above)
Comment (AndriiChuprina) [reply js-arrays/accordion.js]: fixed

File: js-arrays/accordion.js (hunk 8 above)
Comment (AndriiChuprina) [reply js-arrays/accordion.js]: fixed

File: js-arrays/index.html [hunk 12]
```diff
@@ -0,0 +1,53 @@
...
+    <div class="panel">
+        <input type="text" id="element-join"></input>
+        <button onclick="addElementToArray();">Add</button>
+        <button onclick="addSeparator();">Separator</button>
+        <button onclick="displayArray();">Display</button>
+        </br><span>Result array:</span><span id="result-element-join"></span>
+    </div>
+    <button class="accordion">Sum of two</button>
+    <div class="panel">
+        <input type="text" id="element-sum"></input>
```
Comment (valeriya-reva) [inline js-arrays/index.html]: button and panel should be in the wrapper
base structure of accordion
https://getbootstrap.com/docs/4.0/components/collapse/

File: js-arrays/index.html [hunk 13]
```diff
@@ -0,0 +1,53 @@
...
+            </br><span>2 array:</span><span id="result-sum-2"></span>
+            </br><span>+ array:</span><span id="result-element-sum"></span>
+    </div>
+    <button class="accordion">Move an element</button>
+    <div class="panel">
+        <input type="text" id="element-move"></input>
+        <button onclick="addElementToArrayMove();">Add</button>
+        <button onclick="moveElement();">Move(From,To)</button>
+        </br><span>Before array:</span><span id="move-before"></span>
+        </br><span>After array:</span><span id="move-after"></span>
```
Comment (valeriya-reva) [inline js-arrays/index.html]: Should be:
 div
   span
never use br

File: js-arrays/styles.css [hunk 14]
```diff
@@ -0,0 +1,30 @@
...
+.active,
+.accordion:hover {
+    background-color: #ccc;
+    transform: scale(1.01);
+}
+.panel {
+    padding: 0;
+    background-color: white;
+    align-items: center;
+    text-align: center;
```
Comment (dmitry-darly-solutions) [inline js-arrays/styles.css]: image
looks not very nice

File: js-arrays/styles.css [hunk 15]
```diff
@@ -0,0 +1,30 @@
...
+    max-height: 0;
+    transition: max-height 1s cubic-bezier(0, 1, 0, 1);
```
Comment (dmitry-darly-solutions) [inline js-arrays/styles.css]: animation works only on close

**Review by dmitry-darly-solutions — APPROVED** [review]

**Pull request merged**